config/jest/test.config.js

```diff
@@ -8,19 +8,19 @@ const path = require("path");
 module.exports = {
     rootDir: path.join(__dirname, "../../"),
     transform: {
-        "^.+\\.jsx?$": "<rootDir>/config/jest/test.transform.js",
+        "^.+\\.(j|t)sx?$": "<rootDir>/config/jest/test.transform.js",
     },
     restoreMocks: true,
     resetMocks: true,
     testEnvironment: "jest-environment-node",
-    testMatch: ["<rootDir>/**/*.test.js"],
+    testMatch: ["<rootDir>/**/*.test.ts"],
     setupFilesAfterEnv: [
         "jest-extended/all",
         "<rootDir>/config/jest/test-setup.js",
     ],
     moduleNameMapper: {
         "^@khanacademy/wonder-stuff-(.*)$":
-            "<rootDir>/packages/wonder-stuff-$1/src/index.js",
+            "<rootDir>/packages/wonder-stuff-$1/src/index.ts",
     },
     collectCoverageFrom: [
         "packages/**/*.js",
```

Comment:
Presumably these also need to be .ts files?

Reply:
Yes, good catch.